tox.ini

```diff
@@ -155,7 +155,11 @@ basepython = python3
 skip_install = True
 deps =
     mypy
-    git+https://github.com/Shoobx/mypy-zope
+    mypy-zope
```

Comment:
The mypy-zope change is unrelated, just figured that if I'm changing this I'd check if mypy-zope had done a new release, and turns out they had.